packages/search/src/content/index.ts

```diff
@@ -76,6 +86,33 @@ export class ContentSearchService
   }
 }
 
+/**
+ * Turns a site catalog into a content filters object.
+ *
+ * ```js
+ * import { catalogToContentFilter } from '@esri/hub-search';
+ *
+ * const siteCatalog = {
+ *   groups: ['24ad12457b8c410582f185c46f6896ba'],
+ *   orgId: 'be55891b4'
+ * };
+ *
+ * const filter: IContentSearchFilter = catalogToContentFilter(siteCatalog);
+ * ```
+ *
+ * @param catalog - The site catalog
+ * @returns
+ */
+export function catalogToContentFilter({
```

Comment:
I don't want to export this at this time, see https://devtopia.esri.com/dc/hub/issues/1475#issuecomment-2960329

The params it's expecting are not a catalog. I would call this `siteCatalogToContentFilter()` since it takes `site.data.values.catalog`.

Reply:
actually, this fn doesn't do anything and probably never will if `site.data.values.catalog` is only ever `orgId` and/or `groups`.

Let's drop this fn and just do `request.filter = { ...site.data.values.catalog, ...requst.filter }`

Later we can add a different fn for converting a ["platform" catalog](https://github.com/ArcGIS/Hub/pull/54/files)  to `IContentSearchRequest`  b/c I think some things in `catalog.params` will go in `request.filter` and other things will go in `request.options`